source/help/settings/theme-colors.rst

```diff
@@ -0,0 +1,138 @@
+Sidebar Text Hover BG
+  Background color behind channel names and settings tabs as you hover over them.
+Sidebar Text Active Border
+	Color of the rectangular marker on the left side of the Channels pane or Settings sidebar indicating the active channel or tab.
+Sidebar Text Active Color
+	Text color of the active channel or tab in the Channels pane or Settings sidebar.
+Online Indicator
+	Color of the online indicator appearing next to team members names in the Direct Messages list.
+Away Indicator
+	Color of the away indicator appearing next to team members names in the Direct Messages list when they have had no browser activity for 5 minutes.
+Mention Jewel BG
+	Background color of the jewel indicating unread mentions that appears to the right of the channel name. This is also the background color of the “Unread Posts Below/Above” indicator appearing at the top or bottom of the Channels pane on shorter browser windows.
+Mention Jewel Text
+	Text color on the mention jewel indicating the number of unread mentions. This is also the text color on the “Unread Posts Below/Above” indicator.
+
+Center Channel Styles
+~~~~~~~~~~~~~~~~~~~~~
+
+Center Channel BG
+	Color of the center pane, right-hand sidebar and all dialog window backgrounds.
+Center Channel Text
+	Color of all the text - with the exception of mentions, links, hashtags and code blocks - in the center pane, right-hand sidebar, and dialogs.
+New Message Separator
+	The new massage separator appears below the last read message when you click into a channel with unread messages.
+Mention Highlight BG
+	Highlight color behind your words that trigger mentions in the center pane and right-hand sidebar.
+Mention Highlight Link
+	Text color of your words that trigger mentions in the center pane and right-hand sidebar.
+Code Theme
+	Background and syntax colors for all code blocks.
+
+Link and Button Styles
+~~~~~~~~~~~~~~~~~~~~~~
+
+Link Color
+	Text color of all links, hashtags, teammate mentions, and low priority UI buttons.
+Button BG
+	Color of the rectangular background behind all high priority UI buttons.
+Button Text
+	Text color appearing on the rectangular background for all high priority UI buttons.
+
+Exporting a Custom Theme
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can export a theme by copying the "Theme Code" from the Custom Theme menu.
+
+Go to **Account Settings > Display > Custom Theme > Copy and paste to share theme colors** and copy the theme code to export it.
+
+Importing a Custom Theme
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can import a theme by pasting a "Theme Code" into the Custom Theme menu.
+
+Go to **Account Settings > Display > Custom Theme > Copy and paste to share theme colors** and paste a theme code to import it, then click **Save**.
+
+You can find sample themes below. You can also find `theme codes shared by the community from the Mattermost community forum. <https://forum.mattermost.org/t/share-your-favorite-mattermost-theme-colors/1330>`_
```

Comment:
These won't work if they're copied over though, I think, since they use the old theme vector format